### DISCUSSION TITLE
Sync multiple iNaturalist accounts in the same catalog

### DISCUSSION THREAD

**Discussion by dominictinner** [rcloran/lr-inaturalist-publish#12] [Ideas]
I'm in the situation, that my girlfriend and I want to use the same Lightroom catalog (because Lightroom only allows a single catalog to be synced with Creative Cloud). Currently, the plugin only allows for one user to be logged in at the time. Would it be possible to allow each "Publish Services" a unique iNaturalist user account?

**Comment by rcloran** [comment]
Yes! Do you think you can give the zip file at the bottom of https://github.com/rcloran/lr-inaturalist-publish/actions/runs/6722538387 a try? I think it should work for you. Just extract the contents over the existing plugin directory. Now when you create a new iNaturalist publish service, it should start "logged out".

If you don't run into any problems with it, I'll put up a new release with that change soon. If you have any trouble getting that build installed, I'll just do some more thorough testing here and get that out in a release.

I'm also looking for some help testing some Windows-specific changes. If you happen to use Windows, the details are at the bottom of https://github.com/rcloran/lr-inaturalist-publish/commit/4cf60d08367bd6711dca4e506871e366d24893f7 and I could use the help testing!

**Reply by rcloran** [reply]
I'm wondering whether a new publish service should set all the settings (eg, keyword handling) back to defaults instead of using the existing account settings. Setting up a new second account should be an uncommon operation, so I don't think it's super important, but I think it's worth doing now if it makes sense.

What do you think? Are you likely to use different keyword roots (or any other settings) between users, or do you think keeping existing account's settings makes more sense?

**Comment by rcloran** [comment]
I've released 0.6.0 with this change -- you should be prompted to update next time you open Lightroom, or you can check for updates from the plugin manager. Please let me know if you run into problems!
